src/SFA.DAS.Commitments.Api.IntegrationTests/Helpers/TestSetup.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SFA.DAS.Commitments.Api.IntegrationTests.ApiHost;
+using SFA.DAS.Commitments.Api.IntegrationTests.DatabaseSetup;
+using SFA.DAS.Commitments.Api.IntegrationTests.Helpers;
+
+namespace SFA.DAS.Commitments.Api.IntegrationTests.Tests
```

Comment:
Namespace doesn't match location like in other classes. Should be 'SFA.DAS.Commitments.Api.IntegrationTests.Helpers'

Reply:
That's because the OneTimeSetUp & OneTimeTearDown in the TestFixture applies to the namespace it's contained in.